projects/ngx-maplibre-gl/src/lib/markers-for-clusters/markers-for-clusters.component.ts

```diff
@@ -71,30 +71,28 @@ let uniqId = 0;
  * Note: Only use this if you **really** need to use HTML/Angular component to render your symbols. This is **slower** than rendering symbols in WebGL.
  */
 @Component({
-  selector: 'mgl-markers-for-clusters',
+  selector: "mgl-markers-for-clusters",
 
-  template: `
-    <mgl-layer
+  template: ` <mgl-layer
       [id]="layerId"
       [source]="source()"
       type="circle"
       [paint]="{ 'circle-radius': 0 }"
     ></mgl-layer>
-    @for (feature of clusterPoints(); track $index) { 
-      @if (feature.properties.cluster) {
-        <mgl-marker [feature]="feature">
-          <ng-container
-            *ngTemplateOutlet="clusterPointTpl(); context: { $implicit: feature }"
-          ></ng-container>
-        </mgl-marker>
-      } @else {
-        <mgl-marker [feature]="feature">
-          <ng-container
-            *ngTemplateOutlet="pointTpl(); context: { $implicit: feature }"
-          ></ng-container>
-        </mgl-marker>
-      } 
-    }`,
+    @for (feature of clusterPoints(); track $index) { @if
```

Comment:
Is this your IDE setting that is setting the "for" and the "if" in the same line?
I don't think it adds readability...

Reply:
I also have the same formatting issue with VSCode in this project. It wont format the new Angular Control Flow syntax correctly and also changes single quotes to double for me as well. How to you format the code? 

I usually use prettier for formatting

Reply:
I usually prefer lint, although usually not as powerful, it gets the job done.